pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1941,5 +1941,8 @@ func getDomainCreateFlags(vmi *v1.VirtualMachineInstance) libvirt.DomainCreateFl
 	if vmi.ShouldStartPaused() {
 		flags |= libvirt.DOMAIN_START_PAUSED
 	}
+	if vmi.IsCPUDedicated() && vmi.Spec.Domain.CPU.IsolateEmulatorThread {
+		flags |= libvirt.DOMAIN_START_PAUSED
```

Comment:
When do we unpause the VMI in such scenario? Or should the end user do that?

Reply:
RT VMs are those with isolated emulator thread

Reply:
Correct. So nothing to change here, right?

Reply:
Ah sorry, right. This does make sense not only for RT.